projects/containernetworking/plugins/README.md

```diff
@@ -0,0 +1,35 @@
+## CNI Plugins
+
+| Release | Version                                                      |
+|---------|--------------------------------------------------------------|
+| 1-18    | ![Version](https://img.shields.io/badge/version-v0.8.7-blue) |
+| 1-19    | ![Version](https://img.shields.io/badge/version-v0.8.7-blue) |
+| 1-20    | ![Version](https://img.shields.io/badge/version-v0.8.7-blue) |
+| 1-21    | ![Version](https://img.shields.io/badge/version-v0.8.7-blue) |
+| 1-22    | ![Version](https://img.shields.io/badge/version-v0.8.7-blue) |
+
+
+### Updating
+
+1. Work with EKS teams to decide on a new version. Review 
+   [releases](https://github.com/containernetworking/plugins/releases) on the
+   `containernetworking/plugins` [repo](https://github.com/containernetworking/plugins).
+   Please review changelogs carefully looking for updates that may affect EKS-Distro 
+   or downstream projects like EKS-Anywhere, especially if the new version is greater than the one used in
+   upstream Kubernetes.
+2. Update the `GIT_TAG` file to have the new, desired version based on the
+   `plugins` release tags.
+3. Compare the old tag to the new one, looking specifically for Makefile changes.
+   For example:
+   [v0.8.7 compared to v0.9.0](https://github.com/containernetworking/plugins/compare/v0.8.7...v0.9.0). 
+   Check the [scripts/release](https://github.com/containernetworking/plugins/blob/main/scripts/release.sh) any build flag changes, tag changes,
+   dependencies, etc.
+4. Verify the Golang version has not changed. The version specified in
+   [scripts/release](https://github.com/containernetworking/plugins/blob/main/scripts/release.sh) should be the 
+   source of truth. Be sure to select the correct branch for the release when 
+   checking the Golang version. If the version has changes, update 
+   `GOLANG_VERSION` in the release branch.
```

Comment:
Do we want to go ahead and break these versions apart and add  `GOLANG_VERSION` in each release branch directory? I'm guessing that'll eventually be a thing